app/controllers/users.py

```diff
@@ -1195,8 +1216,8 @@ def post(self, user_id):
             log_activity('User', status='Failed',
                          operation='Enable',
                          description=err.body,
-                         a_project_id=project.id,
-                         a_cluster_id=project.cluster_id)
+                         
```

Comment:
Great catch on this, but add affected `user_id`
```suggestion
                         a_user_id=user.id
```

Reply:
Please add this

Reply:
Please add this user id info to the logs